pkg/store/proxy.go

```diff
@@ -256,10 +301,21 @@ func (s *ProxyStore) Series(r *storepb.SeriesRequest, srv storepb.Store_SeriesSe
 				continue
 			}
 
+			var storeTypeStr string
+			if st.StoreType() != nil {
+				storeTypeStr = st.StoreType().String()
+			}
+			metrics := &storeRequestMetrics{
+				withPayload:       s.metrics.timeToFirstByte.WithLabelValues(st.LabelSetsString(), storeTypeStr, withPayloadLabel),
+				withoutPayload:    s.metrics.timeToFirstByte.WithLabelValues(st.LabelSetsString(), storeTypeStr, withoutPayloadLabel),
+				frameTimeoutCount: s.metrics.frameTimeoutCount.WithLabelValues(st.LabelSetsString(), storeTypeStr),
+				queryTimeoutCount: s.metrics.queryTimeoutCount.WithLabelValues(st.LabelSetsString(), storeTypeStr),
+			}
+
+			seriesSet = append(seriesSet, startStreamSeriesSet(seriesCtx, s.logger, closeSeries,
```

Comment:
This should be under the comment, no? (:

Reply:
No, It's not to be commented. Maybe you mean add a comment?

Reply:
There is a comment on the 317 line:
```
 			// Schedule streamSeriesSet that translates gRPC streamed response 
...
```
Could this be moved under that comment or removed?

Reply:
Moved under the comment